tests/test_manual_connect.c

```diff
@@ -160,36 +160,72 @@ void test_mfa_connect_with_duo_passcodeInPassword(void** unused)
     snowflake_term(sf);
 }
 
+void test_okta_connect(void** unused)
```

Comment:
Besides manual function test, can we have unit test cases in automation test? giving test cases in [ODBC](https://github.com/snowflakedb/snowflake-odbc/blob/master/Tests/UnitTests/RestRequestTest/RestRequestTest.cpp#L375) and [libsfclient](https://github.com/snowflakedb/libsnowflakeclient/blob/master/tests/test_unit_oauth.c) ones you made as examples.

Reply:
I made testing cases in the test_unit_okta.cpp